src/app/tracing/components/main-tracing.component.html

```diff
@@ -6,19 +6,29 @@
     >
         <mat-sidenav-content class="mat-sidenav-content-container">
             <div id="mainContent">
-                <fcl-graph-empty [hidden]="(isModelLoaded$ | async) === true" />
+                <fcl-graph-empty
+                    [hidden]="
+                        (isModelLoaded$ | async) === true ||
+                        (isModelLoading$ | async) === true
+                    "
+                />
+                <fcl-graph-loading
+                    [hidden]="(isModelLoading$ | async) === false"
+                />
                 <fcl-schema-graph
                     #schemaGraph
                     [hidden]="
                         (isModelLoaded$ | async) === false ||
-                        (showSchemaGraph$ | async) === false
+                        (showSchemaGraph$ | async) === false ||
+                        (isModelLoading$ | async) === true
```

Comment:
I would not make the graph hidden property dependent on `isModelLoading`, because if you have an loaded graph the graph might vanish for a short time and than come back if the load fails.